packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -44,12 +44,21 @@ interface TabPanelProps extends Omit<AriaTabPanelProps, 'children' | 'style' | '
   children?: ReactNode
 }
 
+const tabPanel = style({
+  marginTop: 4,
+  color: 'gray-800',
+  flexGrow: 1,
+  flexBasis: '[0%]',
```

Comment:
tab panel should start from 0 and grow to whatever size is needed
this is the same in both orientations
the tablist always takes up the exact amount of space it needs, though a user should be able to use style dimension overrides to change the values if they want to explicitly set them or change the minimum/maximum